conan/cli/commands/create.py

```diff
@@ -62,6 +66,13 @@ def create(conan_api, parser, *args):
     lockfile = conan_api.lockfile.update_lockfile_export(lockfile, conanfile, ref, is_build)
 
     print_profiles(profile_host, profile_build)
+    if profile_host.runner and not os.environ.get("CONAN_RUNNER_ENVIRONMENT"):
+        return {
+            'docker': DockerRunner,
+            'ssh': SSHRunner,
+            'wsl': WSLRunner,
+        }[profile_host.runner.get('type')](conan_api, 'create', profile_host, profile_build, args, raw_args).run()
```

Comment:
A bit too much, I wouldn't mind something in 3 separate lines, and also something that raise a controlled ConanException if the ``runner.type`` is not defined or incorrect.